### ISSUE TITLE
Game being counted twice.

### ISSUE THREAD

**Issue by AustinYQM** [extesy/DeckTracker#13]
Running into strange issue where I have 1 game with a deck and that one game is registering as two deck types.

[Here is an image of the tracker.](https://puu.sh/wgkYY/2f1d806afd.png)

Here is my current Decktypes.txt for those two deck types.

```
||Multicolored (FJS)|Colors.Contains("Fire","Justice","Shadow")
|||Infinite Gunsmiths|Cards.ContainsAny(2,"Cabal Spymaster","West-Wind Herald","Calderan Gunsmith")
|||Armory|Cards.ContainsAny(3,"Starsteel Daisho","Icaria, the Liberator","Smuggler's Stash","Rakano Artisan","Quarry","Throne Warden","Harsh Rule","Rise to the Challange","Auric Runehammer")
```

**Comment by AustinYQM** [comment]
[Judging by this tab it is happening with more than just that deck.
](https://puu.sh/wglbi/5bcdbce0ca.png)

**Comment by AustinYQM** [comment]
[Here is an image of the plate match-ups NOT using my custom Decktypes.txt. It still has extra decks. It says I have played 24 but it has 29 listed.](https://puu.sh/wglLb/be77ac07bc.png)

**Comment by jonathanFinnell** [comment]
Not quite related, but shouldn't Infinite Gunsmith be under FPS instead

**Comment by AustinYQM** [comment]
Yeah I've since moved it! Noticed it after I posted those pictures.

**Comment by extesy** [comment]
This is actually by design. It's not counted twice - see that "All Games" is still one? It's because your deck is both Armory _and_ FJS at the same time. The reason is because I wanted to make the right list into a tree view instead and just haven't yet figured it out how to display trees in WPF. With the tree structure all child nodes are aggregated into the parent node so it makes sense. I agree that it looks somewhat confusing in the flat list.

**Comment by extesy** [comment]
For example you want to know your winrate against _all_ FJS decks and not just "all FJS except Armory". So in order to compute this winrate correctly you have to really count all FJS decks, including all subtypes.

**Comment by AustinYQM** [comment]
Gotcha. So it isn't something I have broken. I will continue adding deck types than.

**Comment by extesy** [comment]
Thanks for the pull request! Looks great, I'll add the updated list into the next version.
